src/main/settings.py

```diff
@@ -38,6 +38,7 @@
 
 CSRF_TRUSTED_ORIGINS = ["https://affils.clinicalgenome.org"]
 
+ATOMIC_REQUESTS = True
```

Comment:
I think this should go in the `DATABASES['default']` block.

Reply:
You are correct! I read the documentation incorrectly. Fixed!